internal/server/http/handler/v1/vehicle/handler.go

```diff
@@ -30,13 +31,14 @@ var (
 // @Router			/v1/vehicle [get]
 // @Param			page	query	string	false	"Page"
 // @Param			limit	query	string	false	"Limit"
```

Comment:
Page and limit are integers, right? We can specify types in Swagger 

```suggestion
// @Param			page	query	int	false	"Page"
// @Param			limit	query	int	false	"Limit"
```